### PR TITLE
Add demo docs and resources

### PR DESCRIPTION
**Pull request by ejseqera**
Adds a README to guide users through demonstration on seqeralabs/showcase workspace to add a pipeline, add a dataset, launch, view the run, and view the optimized configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,107 @@
-# `seqeralabs/showcase` Workspace on Nextflow Tower
+# `seqeralabs/showcase` Workspace on Seqera Platform
 
 ## Infrastructure as code
 
-Most [Nextflow Tower](https://cloud.tower.nf/) entities such as Pipelines, Compute Environments and Datasets can be exported in JSON format via the [Nextflow Tower CLI](https://github.com/seqeralabs/tower-cli#nextflow-tower-cli). This is very useful for creating infrastructure as code to store the exact configuration options used to create these entities, and to share and track changes over time.
+Infrastructure as Code (IaC) provides the ability to provision and manage infrastructure through configuration files. Having IaC enables automation in set up, cconsistency and standardization in infrastructure, and documentation of how resources are configured.
```
Comment (laramiellindsey) [inline README.md]:
```suggestion
Infrastructure as Code (IaC) provides the ability to provision and manage infrastructure through configuration files. Having IaC enables automation in set up, consistency and standardization in infrastructure, and documentation of how resources are configured.
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,107 @@
...
-- [Compute Environments](compute-envs)
-- [Pipelines](pipelines)
-- [Datasets](datasets)
+This guide describes how to create resources on Seqera Platform through IaC using two methods: (1) Seqera Platform CLI, and (2) seqerakit.
 
-## Scripts
+### 1. Using Seqera Platform CLI
 
-The bash scripts required to programatically import and export these entities into Tower via the Tower CLI have also been provided in the [`scripts`](scripts) directory. 
+Most [Seqera Platform](https://seqera.io/platform/) entities such as Pipelines, Compute Environments and Datasets can be exported in JSON format via the [Seqera Platform CLI](https://github.com/seqeralabs/tower-cli#nextflow-tower-cli). This is very useful for creating infrastructure as code to store the exact configuration options used to create these entities, and to share and track changes over time.
```
Comment (laramiellindsey) [inline README.md]:
```suggestion
Most [Seqera Platform](https://seqera.io/platform/) entities such as Pipelines, Compute Environments and Datasets can be exported in JSON format via the [Seqera Platform CLI](https://github.com/seqeralabs/tower-cli#nextflow-tower-cli). This is very useful for creating IaC to store the exact configuration options used to create these entities, and to share and track changes over time.
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,107 @@
...
+conda install seqerakit
+```
+
+Or:
+
+```bash
+pip install seqerakit
+```
+
+Please refer to the package's [installation guide](docs/installation.md) for more details on the required dependencies.
```
Comment (laramiellindsey) [inline README.md]: The link to the installation guide is broken.

File: README.md [hunk 4]
```diff
@@ -1,27 +1,107 @@
...
+
+#### Running `seqerakit`
+
+Similar to executing the scripts above to import and create Pipelines, Compute Environments, and Datasets that are defined in JSON onto Seqera Platform, the [YAML files](#using-seqerakit) provided in this repository can also be used create `seqeralabs/showcase` entities on the Platform. For example:
```
Comment (laramiellindsey) [inline README.md]:
```suggestion
Similar to executing the scripts above to import and create Pipelines, Compute Environments, and Datasets that are defined in JSON onto Seqera Platform, the [YAML files](#using-seqerakit) provided in this repository can also be used to create `seqeralabs/showcase` entities on the Platform. For example:
```

File: demo/README.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+This guide provides a walkthrough of the Seqera Platform demonstration. The demonstration will describe how to add a pipeline to the Launchpad, launch a workflow with pipeline parameters, monitor a run, and examine the run details.
+
+More specifically, this demonstration will focus on using the [nf-core/rnaseq](https://github.com/nf-core/rnaseq) pipeline on AWS Batch.
+
+
+## Requirements
+* AWS Compute Environment
+* [nf-core/rnaseq](https://github.com/nf-core/rnaseq)
+* Dataset added for minimal test RNAseq data (see [samplesheet_test.csv](/samplesheet_test.csv) file)
```
Comment (laramiellindsey) [inline demo/README.md]: The link to the samplesheet_test.csv is broken.

**Review by laramiellindsey — COMMENTED** [review]
Looks good! Minor changes/suggestions. I think it covers everything needed in a demo and the structure is easy to follow.

File: demo/README.md (hunk 5 above)
Comment (joshiesh) [reply demo/README.md]: Good catch, will fix. Thanks Laramie.

File: demo/demo_walkthrough.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+
+Log into Seqera Platform, either through a GitHub account, Google account, or an email address.
+
+If an email address is provided, Seqera Cloud will send an authentication link to the email address to login with.
+
+![Seqera Platform Cloud login](docs/images/sp-cloud-signin.gif)
+
+### 2. Navigate into the seqeralabs/showcase Workspace
+
+All resources in Seqera Platform live inside a Workspace, which in turn belong to an Organisation. Typically, teams of colleagues or collaborators will share one or more workspaces. All resources in a Workspace (i.e. pipelines, compute environments, datasets) are shared by members of that workspace.
```
Comment (robsyme) [inline demo/demo_walkthrough.md]:
```suggestion
All resources in Seqera Platform live inside a Workspace, which in turn belong to an Organisation. Typically, teams of colleagues or collaborators will share one or more workspaces. All resources in a Workspace (i.e. pipelines, compute environments, datasets) are shared by members of that workspace.

The workspace can be divided into two "halves" - a backend half for managing the workspace and infrastructure (using the "Credentials", "Compute Environments", "Secrets" and "Participants" tabs) and a frontend half for launching and monitoring Nextflow runs. 

The backend functions are of use to a cloud administrator and the frontend functions by a bench or computational biologist.
```

File: demo/demo_walkthrough.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+### 13. Optimize the Pipeline
+
+When a run completes successfully, an optimized profile is created. This profile consists of Nextflow configuration settings for each process and each resource directive (where applicable): cpus, memory, and time. The optimized setting for a given process and resource directive is based on the maximum use of that resource across all tasks in that process.
+
+Navigate back to the Launchpad, click on the nf-core/rnaseq Pipeline added, and click on the 'Lightbulb' icon to view the optimized profile.
+
+![Optimized configuration](docs/images/optimize-configuration.gif)
+
+**Congratulations!**
+You just added a pipeline to the Launchpad, and executed it on Seqera Platform! 🚀
```
Comment (robsyme) [inline demo/demo_walkthrough.md]:
```suggestion
You just added a pipeline to the Launchpad, and executed it on Seqera Platform! 🚀

###  14. Credentials (optional for demo)

To execute each run, a job is submitted to your executor. Be it cloud or HPC, the Seqera Platform will require some credentials to submit the job on your behalf. An Admin user can add these keys to the "Credentials" on behalf of the other workspace users. The "Credentials" tab can also be used to add keys and tokens for container registries or Git hosting providers. Our demonstration so far has pulled public data and public workflows, but the Seqera Platform can just as easily be used to execute private workflows using private containers.


### 15. Compute Environments (optional for demo)

As introduced earlier - the Seqera Platform does not host data nor perform the computation - it's a lens into your own infrastructure where the data and computation always remain. Before any runs can be submitted in a workspace, an Admin user needs to add one or more Compute Environments to the workspace. The Compute Environment creation has two paths - a "manual" mode to add pre-existing infrastructure to the workspace, and a "forge" mode where the Seqera Platform itself creates the infra in your cloud account necessary to run Nextflow workloads.

If you have an experienced cloud team, or systems administrators who have already built and administer your cloud or HPC infrastructure, then you only need provide the queue names to which the jobs can be submitted. If you would prefer the platform to set up the infrastructure on your behalf, the user is provided more options to customize the final compute environment. 

Add a CE is done once for the workspace. It takes only a few minutes and is then available for all users in the workspace as an environment in which they can execute their runs.
```

**Review by robsyme — CHANGES_REQUESTED** [review]
I always try to highlight the CE creation step (particularly for smaller groups). Added some example copy for credentials and CEs.

File: demo/demo_walkthrough.md (hunk 7 above)
Comment (joshiesh) [reply demo/demo_walkthrough.md]: I agree, I tend to highlight these things too (time permitting). I was going to add the CE creation, credentials, overview of secrets as separate markdown docs (the main walkthrough doc is getting quite lengthy) - thoughts?

**Pull request closed without merging**